w3-hw1-ThuyDung/cypress/pages/bookApplication/bookPage.js

```diff
@@ -0,0 +1,36 @@
+export const bookPage= {
+    TXT_SEARCH : "#searchBox",
+    BTN_BOOK :".action-buttons",
+    HEADER: ".main-header",
+    LBL_BOOK: "#title-wrapper > div.col-md-9.col-sm-12 label",
+    BTN_ADDBOOK : "div.text-right.fullButton",
+
+    isNavigateToBookStore(){
+        cy.get(this.HEADER).should('have.text', "Book Store");
+        return this;
+    },
+    findBookByTyping(name){
+        cy.get(this.TXT_SEARCH).type(name);
+        cy.get(this.BTN_BOOK).contains(name).eq(0).click();
+        return this;
+    },
+    isBookCorrect(name){
```

Comment:
You can skip the verify here, it is for feature `See the book detail` or something like that